changelogs/current.yaml

```diff
@@ -18,6 +18,9 @@ behavior_changes:
 
 minor_behavior_changes:
 # *Changes that may cause incompatibilities for some users, but should not for most*
+- area: quic
```

Comment:
I thought that at first, but if someone has made a third-party extension on [envoy.quic.connection_id_generator](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/listener/v3/quic_config.proto#extension-category-envoy-quic-connection-id-generator) they're *probably* going to be calling this function from that extension, and the change would break their build and require them to update their extension.

(I was tempted to include the typo function name as an alias of the corrected function name, but I couldn't find the right way to mark such a thing deprecated!)

Reply:
My understanding is that the util function modifies the first [some number] of bits to help envoy route things associated with the connection ID. It seems like *not* using it in a custom id generator would likely result in things not working properly, or at least not working optimally.
Our model for this would be envoy "owns" enough bits to support its needs, then we'd use some number of custom bits after that in support of a load balancer, so I think our currently-hypothetical extension would still be calling this util function.

Reply:
This is only used in the Envoy default implementation of the EnvoyQuicConnectionIdGeneratorFactory extension. And Envoy doesn't commit to stable behavior of it. The extension interface covers both how Envoy generate CIDs and how it routes the packet across worker threads. If you are going to implement a new one, it's better to override both, instead of mixing the Envoy default routing behavior with your own CID generation logic.

Reply:
Fair enough, I've reverted the changelog change then. Glad I made this mistake so I learned this about the extension, thanks! :)